README.md

```diff
@@ -61,15 +61,15 @@ The 'output' is your encrypted password. This encrypted password can then be use
         #Encrypted Database Password
         password: ENC(EHL/foiBKY2Ucy3oYmxdkFiXzWnOu7by)
             
-Finally if you are using a encrypted password, when you run Circus Train you must provide the --password= parameter, providing your decryption password as an argument.
-In this example you would run Circus Train Housekeeping as follows:
+Finally if you are using a encrypted password, when you run  you must provide the --password= parameter, providing your decryption password as an argument.
```

Comment:
Since we don't have a shell script none of this really makes sense. Remove for now.